contrib/rpm/buildah.spec

```diff
@@ -97,7 +97,7 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 %{_datadir}/bash-completion/completions/*
 
 %changelog
-* Wed Nov 21 2018 Tom Sweeney <tsweeney@redhat.com> 1.6-dev-1
+* Tues Jan 15 2019 Tom Sweeney <tsweeney@redhat.com> 1.6-1
```

Comment:
`rpm -q --specfile` on this gives me an error.  Using a three-letter version of the day-of-the-week (i.e., changing `Tues` to `Tue`) makes it acceptable.

Reply:
rats and thanks @nalind, will fix that.